templates/includes/base/header.html

```diff
@@ -1,32 +1,33 @@
+          developer
+        </a>
+    </div>
 
-        <button class="header-search__toggle" onclick="document.getElementById('search').classList.toggle('is-visible'); document.getElementById('navigation').classList.remove('is-visible')">
-            <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image" alt="Perform a search" />
-        </button>
+    <button class="p-navigation__toggle" onclick="document.getElementById('navigation').classList.toggle('is-visible'); document.getElementById('search').classList.remove('is-visible')">
+        <img src="{{ ASSET_SERVER_URL }}12387180-navigation-menu-plain.svg" class="p-navigation__icon" alt="Expand/collapse menu" />
+    </button>
 
-        <nav id="navigation" class="p-navigation__nav">
-            <span class="u-off-screen">
-                <a href="#main-content">Jump to main content</a>
-            </span>
-            <a class="p-navigation__my-apps-link" href="https://myapps.developer.ubuntu.com/">My Apps</a>
-        </nav>
+    <button class="header-search__toggle" onclick="document.getElementById('search').classList.toggle('is-visible'); document.getElementById('navigation').classList.remove('is-visible')">
+        <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image" alt="Perform a search" />
+    </button>
 
-        <form id="search" action="/search" class="header-search">
-            <label for="edit-keys" class="header-search__label">Search</label>
-            <input class="header-search__field" type="search" name="q" id="search" placeholder="Search the site" />
-            <button type="submit" class="header-search__submit">
-                <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image--large-screen" alt="Perform a search" />
-                <img src="{{ ASSET_SERVER_URL }}19750a6d-search-black.svg" class="header-search__image--small-screen" alt="Perform a search" />
-            </button>
-        </form>
-    </div>
+    <nav id="navigation" class="p-navigation__nav">
+        <span class="u-off-screen">
+            <a href="#main-content">Jump to main content</a>
+        </span>
+    </nav>
+
+    <form id="search" action="/search" class="header-search">
+        <label for="edit-keys" class="header-search__label">Search</label>
+        <input class="header-search__field" type="search" name="q" id="search" placeholder="Search the site" />
+        <button type="submit" class="header-search__submit">
+            <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image--large-screen" alt="Perform a search" />
+            <img src="{{ ASSET_SERVER_URL }}19750a6d-search-black.svg" class="header-search__image--small-screen" alt="Perform a search" />
```

Comment:
These were already in place, I think they’re just white space changes, in the interest of MVP would it be okay for me to add the improvement as an issue?